README.md

```diff
@@ -5,6 +5,112 @@
         <img width="30%" src="https://sulu.io/uploads/media/800x/00/230-Official%20Bundle%20Seal.svg?v=2-6&inline=1" alt="Official Sulu Bundle Badge">
     </a>
 </p>
+### Upgrading Data from Sulu 2.6 to Sulu 3.0
+
+The upgrade from Sulu 2.6 to Sulu 3.0 is a major upgrade and will require some migration steps.
+
+Due to the fact that in Sulu 3.0 the SuluArticleBundle is merged into the core Sulu package, before updating the 
+sulu/sulu dependency, you have to update the sulu/article-bundle dependency to the latest version. 
```

Comment:
Thinking about this sentence, it sounds like you always have to update the article bundle, but this is only required if you had installed the SuluArticleBundle. What do you think about the following? @alexander-schranz @mamazu 

```suggestion
Due to the fact that in Sulu 3.0 the SuluArticleBundle is merged into the core Sulu package, if you have the sulu/article-bundle installed, you need to update the sulu/article-bundle dependency to the latest version before updating the sulu/sulu dependency. If the sulu/article-bundle is not installed, this step is not necessary.
```